cime_config/testdefs/testlist_clm.xml

```diff
@@ -150,7 +150,7 @@
       <machine name="izumi" compiler="nag" category="aux_clm"/>
     </machines>
     <options>
-      <option name="wallclock">00:30:00</option>
```

Comment:
Hmmm. You shouldn't have had to make this change, because the run time is only 535 seconds. It is up from 441 seconds for the ctsm5.3.059. So I'm curious what the fail looked like for you. It's not necessarily bad to bump it up slightly either though...

Reply:
Resolving in prep for merge.